app/coffee/RequestParser.coffee

```diff
@@ -31,6 +31,12 @@ module.exports = RequestParser =
 			response.check = @_parseAttribute "check",
 				compile.options.check,
 				type: "string"
+			response.incremental = @_parseAttribute "incremental",
+				compile.options.incremental,
+				type: "string"
+			response.state = @_parseAttribute "state",
+				compile.options.state,
+				type: "string"
```

Comment:
Can you document on the CLSI side what the different 'states' are? Perhaps do a check here that it matches one of a set of known states. Mainly from a documentation point of view, since my understanding is the only thing that really matters is whether it has changed, but knowing what things to expect helps to build up a model of how it will behave in practice.